src/installer/pkg/archives/dotnet-nethost.proj

```diff
@@ -0,0 +1,40 @@
+<Project Sdk="Microsoft.Build.NoTargets">
+  <PropertyGroup>
+    <CreateSymbolsArchive Condition="'$(DotNetBuildFromSource)' != 'true'">true</CreateSymbolsArchive>
```

Comment:
Isn't this needed for sourcebuild and why wouldn't it work for sourcebuild?

Reply:
The source-build build for the hosts doesn't strip symbols, so there's no symbol file to archive.

Reply:
Ah we have a similar case in the System.IO.Ports native package: https://github.com/dotnet/runtime/blob/b9eb5acbde03e0b7ddda6a16443ef4490a19aced/src/libraries/System.IO.Ports/pkg/runtime.native.System.IO.Ports.props#L11-L12. I think it would make more sense to use that condition and add a comment here as well.

Reply:
```suggestion
	<!-- When KeepNativeSymbols is set, debug symbols are kept in the .so files. So when it is true, we don't need a symbols archive. -->
    <CreateSymbolsArchive Condition="'$(KeepNativeSymbols)' != 'true'">true</CreateSymbolsArchive>
```